src/OneBlinkFormBase.tsx

```diff
@@ -82,6 +86,11 @@ export type OneBlinkReadOnlyFormProps = {
    * scheduled task associated with a group via a form submission
    */
   taskGroupInstance?: ScheduledTasksTypes.TaskGroupInstance
+  /**
+   * Override the default behaviour when injecting values, such as in an info
+   * element or a default value.
+   */
+  replaceInjectablesOverrides?: ReplaceInjectablesOverrides
```

Comment:
More react-y to make this `userProfile` so that we can re-render when the object changes. There are couple of other places we use the `userProfile` from `useAuth()`. We will need to ensure we not using `useAuth()` for any user profile data.

Reply:
Well thats sort of the point. I don't want to replace the current logged in user everywhere. Only in the replaceable value stuff.

Reply:
Anything else I would consider a side effect.

Reply:
Although, it looks like the only other place using `userProfile` is the dynamic option set replaceable stuff, which does the seem thing but in its own way. so that will need to update too.

Reply:
The other touch point we will need to cater for is injecting dynamic options into options based form elements. Im not sure if that is catered for already with the code change you have made so far.

Reply:
Agreed.

Reply:
Does seem strange to require a function when we don't really have a use case for passing a function but I am ok with it if you would prefer a function.